tests/docker-compose.yml

```diff
@@ -15,11 +17,13 @@ services:
       start_period: 5s
     command: tail -f /dev/null
 
-  shouldnt-restart:
+  shouldnt-restart-healthy:
+    # this container shouldn't be restarted by autoheal because its healthy
     image: alpine
-    container_name: shouldnt-restart
     network_mode: none
     restart: "no"
+    labels:
+      - "$AUTOHEAL_CONTAINER_LABEL=true"
```

Comment:
should there be `$` here ?

Reply:
This enables to define unique labels (autoheal_label_12345=true and autoheal_label_12346=true) for different CI runs also ensuring a not properly cleaned up test does not interfere with another test (when unique labels are used)

Reply:
all makes sense now 👍